### DISCUSSION TITLE
Firefox 98.0.1 详情页面显示异常

### DISCUSSION THREAD

**Discussion by nukemiko** [Gamuxorg/bbs#361] [游戏或网站运行反馈]
在排障模式下测试的结果：
![Screenshot_20220317_215528](https://user-images.githubusercontent.com/53572814/158825426-783a86f8-16c5-46f0-822d-1d65daffc1f5.png)

**Comment by hyh19962008** [comment]
你有没有安装插件或者油猴脚本啊？同FF98没有问题

**Comment by nukemiko** [comment]
> 你有没有安装插件或者油猴脚本啊？同FF98没有问题

在排障模式下开的页面，包括油猴在内的所有插件都不能用，所以不会是我装的插件的问题

**Comment by shouhuanxiaoji** [comment]
系统信息发一下吧。

**Comment by nukemiko** [comment]
> 系统信息发一下吧。

不好意思啊，刚刚看过网站页面，这个问题好像修复了。

系统信息：https://pst.klgrth.io/paste/85aepuzft4na9bm73wkhn9xm

**Comment by shouhuanxiaoji** [comment]
感谢测试
